.github/workflows/test.yml

```
@@ -0,0 +1,24 @@
+name: Run tests
+
+on: [push, pull_request]
+
+jobs:
+  test:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        ghidra: ["9.2", "9.2.1", "9.2.2", "9.2.3", "9.2.4"]
+    steps:
+      - uses: actions/checkout@v1
```

```suggestion
      - uses: actions/checkout@v2
```
if you want all commits:
```suggestion
      - uses: actions/checkout@v2
        with:
          fetch-depth: 0
```